e2e/app/page-identifiers.ts

```diff
@@ -16,8 +22,34 @@ export enum EllipsisMenuAction {
    Rename = 'Rename',
 }
 
-export enum PageUrl {
-   Home = config.uiBaseUrl,
-   Workspaces = config.uiBaseUrl + config.workspacesUrlPath,
-   Admin = config.uiBaseUrl + config.adminUrlPath,
+export enum LinkText {
+   Confirm = 'Confirm',
+   KeepEditing = 'Keep Editing',
+   Cancel = 'Cancel',
+   Calculate = 'Calculate',
+   AddThis = 'ADD THIS',
+   Finish = 'Finish',
+   Rename = 'Rename',
+   Save = 'Save',
+   Update = 'Update',
+   Next = 'Next',
+   CreateSet = 'CREATE SET',
```

Comment:
Not related to this PR but curious why this link/Button is all in uppercase and not consistent with other links